src/main/scala/viper/silver/frontend/SilFrontend.scala

```diff
@@ -55,7 +56,9 @@ trait SilFrontend extends DefaultFrontend {
   def resetPlugins(): Unit = {
     val pluginsArg: Option[String] = if (_config != null) {
       // concat defined plugins and default plugins
-      val list = _config.plugin.toOption ++ (if (_config.disableDefaultPlugins()) Seq() else defaultPlugins)
+      val list = _config.plugin.toOption ++
+        (if (_config.enableSmokeDetection()) Seq("viper.silver.plugin.standard.smoke.SmokeDetectionPlugin") else Seq()) ++
```

Comment:
I guess the name of the plugin could be a separate val field?

Reply:
-> 98dace16